build.sbt

```diff
@@ -129,18 +130,21 @@ val bloop = project
 /***************************************************************************************************/
 /*                      This is the corner for all the command definitions                         */
 /***************************************************************************************************/
-
 val publishLocalCmd = Keys.publishLocal.key.label
-addCommandAlias("setupIntegrations", List(
-  s"+${integrationsCore.id}/$publishLocalCmd",
-  s"^${sbtBloop.id}/$publishLocalCmd",
-  s"${mavenBloop.id}/$publishLocalCmd"
-).mkString(";", ";", ""))
+addCommandAlias(
+  "setupIntegrations",
+  List(
+    s"+${integrationsCore.id}/$publishLocalCmd",
+    s"^${sbtBloop.id}/$publishLocalCmd",
+    s"${mavenBloop.id}/$publishLocalCmd"
+  ).mkString(";", ";", "")
+)
 
-addCommandAlias("runTests", List(
-  s"${sbtBloop.id}/${scriptedAddSbtBloop.key.label}",
-  s"${sbtBloop.id}/${scripted.key.label}"
-).mkString(";", ";", ""))
+addCommandAlias("runTests",
```

Comment:
Can we format this one like `setupIntegrations` :smile:?

Reply:
scalafmt doesn't agree -- do you really want to go against it?

Reply:
@Duhemm http://scalameta.org/scalafmt/#Configstyle

Reply:
```
align = none
danglingParentheses = true
```
gives you this by default, fwiw

Reply:
Actually scalafmt is smart enough to understand what I want to do on its own 😱